src/components/ToolTip/index.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import 'test-util/setup';
+import { shallow } from 'enzyme';
+import { wrapTooltip } from './index';
+
+const buttonText = 'children';
+const defaultProps = {
+  text: 'text',
+  children: <button>{buttonText}</button>,
+};
+
+const setup = (props = {}) => {
+  const testProps = { ...defaultProps, ...props };
+  return shallow(wrapTooltip(testProps.children, testProps.text));
+};
+
+describe('ToolTip', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = setup();
+  });
+
+  describe('Render', () => {
+    it('without errors', () => {
+      expect(wrapper.length).toBe(1);
+    });
+
+    it('children rendered without error', () => {
+      expect(wrapper.find(`.hover-tooltip`).length).toBe(1);
+    });
+
+    it('text rendered without error', () => {
+      expect(wrapper.find(`.hover-tooltip__inner`).length).toBe(1);
```

Comment:
instead of looking for the tooltip text wrapper - check for the `.text()` contents instead. this will reduce the dependency on testing implementation (including knowing classnames)